docs/installation/frameworks/react.md

```diff
@@ -500,7 +500,52 @@ Finally, exclude CKEditor 5 SVG and CSS files from `file-loader`. Find the last
 }
 ```
 
-#### Using CKEditor 5 source
+Now, your setup with `create-react-app` is complete. You can also check how to configure Vite in the next section or move straight to [Using the editor from source](#using-the-editor-from-source).
+
+### Vite
+
+This guide assumes that you use `create-vite` as your boilerplate. To get started with Vite and React, run the command below.
+
+```bash
+# npm 6.x
+npm create vite@latest ckeditor5-react-example --template react
+
+# npm7+, extra double-dash is needed:
+npm create vite@latest ckeditor5-react-example -- --template react
+```
+
+This command will install and execute `create-vite`, the official project scaffolding tool for Vite.
+
+#### Installing necessary packages
+
+Besides the CKEditor base and plugins, you need to install additional packages to use it from source with React and Vite: the Vite plugin, the official React component, and the default theme.
+
+<info-box>
+	Using the Vite plugin to build CKEditor 5 from the source in Vite is still in the experimental phase. We encourage you to test it and give us feedback. To read more about integration with Vite or its limitations, check the {@link installation/advanced/integrating-from-source-vite Integrating from source with Vite} guide.
+</info-box>
+
+Install necessary packages alongside the default theme using the following command.
+
+```bash
+npm install --save @ckeditor/vite-plugin-ckeditor5 @ckeditor/ckeditor5-react @ckeditor/ckeditor5-theme-lark
+```
+
+#### Configuring `vite.config.js`
+
+Configuring CKEditor with React and Vite is simple. Modify the existing config by importing `ckeditor5` and adding it to the list of plugins.
+
+```js
+// vite.config.js
+
+import ckeditor5 from '@ckeditor/vite-plugin-ckeditor5';
+
+export default defineConfig( {
+	plugins: [
```

Comment:
In React apps created by create-vite, there will be `react()` plugin used as well. We can show `vite.config.js` example with that plugin, as this will be only 2 lines more and it will show that ckeditor5() is added along other vite plugins in this config.